multiqc/utils/search_patterns.yaml

```
@@ -325,7 +325,7 @@ eigenstratdatabasetools:
 fastp:
   fn: "*.json"
   contents: '"before_filtering": {'
-  num_lines: 3
+  num_lines: 5
```

I suspect that the difference between 5 and 50 is negligible, so may as well go all-in and avoid the need for future PRs..
```suggestion
  num_lines: 50
```